__tests__/app.test.js

```diff
@@ -175,23 +175,25 @@ describe('GET /api/articles/:article_id/comments', () => {
 
         })
     })
-    test('status:400, responds with "Invalid Input" when article_id is an invalid type', () => {
+    test('status:200, returns an empty array when article_id is valid but does not have any comments', () => {
         return request(app)
-        .get('/api/articles/:not_article_id/comments')
-        .expect(400)
+        .get('/api/articles/11/comments')
+        .expect(200)
         .then(({body}) => {
 
-            expect(body.msg).toBe('Invalid Input')
+            const { comments } = body;
+
+            expect(comments).toHaveLength(0)
 
         })
     })
-    test('status:404, responds with "Not Found" when article_id is a valid type but does not exist', () => {
+    test('status:400, responds with "Invalid Input" when article_id is an invalid type', () => {
         return request(app)
-        .get('/api/articles/9999/comments')
-        .expect(404)
+        .get('/api/articles/:not_article_id/comments')
+        .expect(400)
         .then(({body}) => {
 
-            expect(body.msg).toBe('Not Found')
+            expect(body.msg).toBe('Invalid Input')
 
         })
```

Comment:
Here you have removed the test for when a valid id is given but does not exist, this was a really good test and should be in this ticket.